src/component/books.css

```diff
@@ -97,6 +117,100 @@
   margin-top: 3px;
 }
 
+.progressbar {
+  width: 40%;
+}
+
 .forms {
-  margin: 24px;
+  margin: 0.313rem 6.25rem 0;
+  padding: 1rem 9.188rem 1.625rem 1.688rem;
+  display: inline-block;
+  width: 100%;
+}
+
+form input,
+select {
+  margin: 1.188rem 1.125rem 0 0;
+  padding: 0.813rem 5rem 0.813rem 1.063rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+}
+
```

Comment:
- [ _Optional_ ] Kindly ensure that your form is aligned perfectly on the left with the rest of the components in the Bookstore. Right now it does not follow the same horizontal alignment as in the Zeplin template which you can see below
> Zeplin Design
![image](https://user-images.githubusercontent.com/90556221/180768322-73744a0b-d429-400c-a8ff-464ae66fab5c.png)

> Your Design
![image](https://user-images.githubusercontent.com/90556221/180768516-b86f4aa9-80c6-43a6-a4d8-7d040a968476.png)

- [ _Optional_ ] Additionally, you also seem to be missing the `ADD NEW BOOK` heading in your design as you can see above. Kindly add it so you mimic the Zeplic design perfectly.